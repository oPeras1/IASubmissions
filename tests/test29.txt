1	1	1	1	2	3	3	3	3	4	4	5	6	6	7	7	7	8	8	8
1	1	2	2	2	2	4	4	3	4	4	5	6	7	7	8	8	8	9	9
10	10	2	11	11	11	4	4	4	4	5	5	6	7	8	8	9	9	9	12
10	13	13	13	14	11	11	11	4	4	4	5	5	7	7	8	7	15	15	12
10	13	16	13	14	17	18	18	18	4	4	19	5	7	7	7	7	15	12	12
16	16	16	13	14	17	17	17	18	4	19	19	5	20	20	20	15	15	15	12
21	16	14	14	14	17	18	18	18	4	19	19	5	20	20	20	20	20	20	12
21	21	21	14	14	17	18	22	23	23	23	19	19	24	24	24	24	20	25	12
26	21	21	21	14	14	22	22	23	23	27	27	27	24	24	28	28	25	25	25
26	26	26	21	21	14	29	22	23	23	30	27	30	24	28	28	28	28	28	28
31	26	32	21	33	14	29	29	23	34	30	30	30	24	28	35	36	36	36	36
31	32	32	33	33	33	29	34	34	34	34	37	30	38	38	35	35	35	35	35
31	32	29	29	29	29	29	34	39	34	37	37	37	38	35	35	40	40	35	35
31	32	32	29	29	41	29	29	39	39	37	37	37	38	38	38	38	40	40	38
31	32	42	42	41	41	41	39	39	37	37	43	38	38	44	44	38	38	38	38
31	42	42	45	45	45	41	41	41	41	37	43	43	43	43	44	44	46	46	46
45	45	45	45	47	47	48	48	48	37	37	43	49	49	49	49	49	49	49	46
45	45	45	47	47	50	50	50	48	37	37	51	49	49	52	52	49	49	49	49
53	54	54	54	47	47	50	50	48	37	37	51	51	52	52	49	49	55	55	49
53	53	53	54	54	54	54	50	50	50	37	37	51	52	52	52	55	55	49	49