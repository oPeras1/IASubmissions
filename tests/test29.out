T	T	T	1	T	3	L	L	L	4	4	5	L	L	7	7	7	8	8	8
1	T	2	T	T	T	4	4	L	I	4	5	L	7	7	S	S	8	S	S
L	L	2	11	11	L	4	4	4	I	5	5	L	7	S	S	9	S	S	12
L	13	L	L	14	L	L	L	4	I	4	5	I	7	T	8	7	L	15	I
L	13	16	L	14	T	18	18	L	I	4	19	I	T	T	T	7	L	12	I
T	T	T	L	14	T	T	17	L	4	S	19	I	20	20	20	15	L	L	I
21	T	14	14	L	T	18	L	L	4	S	S	I	20	I	I	I	I	20	I
21	S	S	14	L	17	18	T	23	L	23	S	19	L	L	24	24	20	T	12
L	21	S	S	L	L	T	T	23	L	T	T	T	L	24	S	S	T	T	T
L	L	L	21	21	14	29	T	L	L	30	T	30	L	S	S	28	28	28	28
31	26	S	21	T	14	L	L	23	T	L	L	L	24	28	L	I	I	I	I
31	S	S	T	T	T	L	34	T	T	34	37	L	I	38	L	35	35	35	35
I	S	29	29	29	29	L	34	39	T	S	S	37	I	L	L	S	S	35	35
I	32	32	29	29	T	29	29	S	S	37	S	S	I	38	38	38	S	S	38
I	32	S	S	T	T	T	S	S	37	37	T	38	I	S	S	38	38	38	38
I	S	S	45	45	45	41	41	41	41	37	T	T	43	43	S	S	L	L	L
45	T	45	45	S	S	L	L	L	37	37	T	49	L	L	L	49	49	49	L
T	T	T	S	S	50	50	50	L	37	37	S	49	L	52	52	49	49	49	49
L	54	54	54	47	47	50	L	48	37	37	S	S	52	T	49	49	S	S	49
L	L	L	I	I	I	I	L	L	L	37	37	S	T	T	T	S	S	49	49